snakemake/io.py

```diff
@@ -1517,6 +1517,21 @@ def _insert_items(self, index, items):
             elif i == index:
                 self._set_name(name, i, end=i + len(items))
 
+    def _get_unnamed_arguments(self):
```

Comment:
This should not be private as we want to use it in snakemake wrappers.

Reply:
I would suggest `def get_positional_items(self) -> Sequence[str]:`.

Reply:
Name changed as requested.